source/hic_hal/nxp/lpc11u35/gpio.c

```diff
@@ -81,7 +81,25 @@ void gpio_init(void)
 {
     // enable clock for GPIO port 0
     LPC_SYSCON->SYSAHBCLKCTRL |= (1UL << 6);
+#if defined(TARGET_POWER_HOLD)
+    // Target PowerHOLD port
+    PIN_PWH_IOCON = PIN_PWH_IOCON_INIT;
+    LPC_GPIO->CLR[PIN_PWH_PORT] = PIN_PWH;
+    LPC_GPIO->DIR[PIN_PWH_PORT] |= PIN_PWH;
+#endif
     // configure GPIO-LED as output
+#if defined(CONTROLLED_POWER_LED)
+    // Power led (red)
+    PIN_POW_LED_IOCON = PIN_POW_LED_IOCON_INIT;
+    LPC_GPIO->CLR[PIN_POW_LED_PORT] = PIN_POW_LED;
+    LPC_GPIO->DIR[PIN_POW_LED_PORT] |= PIN_POW_LED;
+#endif
+#if defined(INTEGRATED_STATUS_LED)
+    // DAP/MSD/Serial led (green)
+    PIN_DAP_LED_IOCON = PIN_DAP_LED_IOCON_INIT;
+    LPC_GPIO->SET[PIN_DAP_LED_PORT] = PIN_DAP_LED;
+    LPC_GPIO->DIR[PIN_DAP_LED_PORT] |= PIN_DAP_LED;
```

Comment:
This is the same as "DAP led (green)" below so you should be able to remove "#if defined(INTEGRATED_STATUS_LED)". This will cause PIO0_16 (used for DAP LED, MSD LED, CDC LED on the board) to be initialized to the same value multiple times, but that shouldn't be a problem.